.github/workflows/_build-native-meta.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       with:
         registry: ghcr.io
         username: ${{ github.repository_owner }}
-        password: ${{ secrets.GITHUB_TOKEN }}
+        password: ${{ github.repository_owner == 'DependencyTrack' && secrets.BOT_IMAGE_PUSH_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
This works like a ternary operator, see https://github.com/actions/runner/issues/409#issuecomment-752775072

Idea being that forks of this repository should safely fall back to the default `secrets.GITHUB_TOKEN`.